src/hotspot/share/runtime/threadSMR.hpp

```diff
@@ -69,23 +69,17 @@ class ThreadsList;
 //   }
 //   :  // do stuff with 'jt'...
 //
-// JVM/TI oop example (this one should be very rare):
-//   oop thread_obj = ...;
-//   :
-//   JavaThread *jt = nullptr;
-//   ThreadsListHandle tlh;
-//   jvmtiError err = JvmtiExport::cv_oop_to_JavaThread(tlh.list(), thread_obj, &jt);
-//   if (err != JVMTI_ERROR_NONE) {
-//     return err;
-//   }
-//   :  // do stuff with 'jt'...
-//
-// A JavaThread * that is included in the ThreadsList that is held by
+// A JavaThread* that is included in the ThreadsList that is held by
 // a ThreadsListHandle is protected as long as the ThreadsListHandle
-// remains in scope. The target JavaThread * may have logically exited,
-// but that target JavaThread * will not be deleted until it is no
+// remains in scope. The target JavaThread* may have logically exited,
+// but that target JavaThread* will not be deleted until it is no
 // longer protected by a ThreadsListHandle.
 //
+// Note that for virtual threads, we obtain a reference to the carrier JavaThread
```

Comment:
This is not correct for `JvmtiExport::cv_external_thread_to_JavaThread` (it does not return carrier's JavaThread).
JVMTI has `JvmtiEnvBase::get_threadOop_and_JavaThread` and `JvmtiEnvBase::get_JavaThread_or_null` to get carrier thread